src/Fixer/Alias/RandomApiMigrationFixer.php

```diff
@@ -38,16 +40,24 @@ public function configure(array $configuration = null)
         static $validFunctions = array('rand', 'srand', 'getrandmax');
```

Comment:
again:

> why end user is supposed to provide number of arguments ?

----

> there is BC problem on master right?

OK, no issue as it was not on 1.x, nothing to describe in update guide